server/graphql/programs/src/types/program.rs

```diff
@@ -47,7 +47,7 @@ impl ProgramSortInput {
 }
 
 #[derive(InputObject, Clone)]
-pub struct ProgramFilterInput {
+pub struct programOptionsOrFilterInput {
```

Comment:
I think this is still just a program filter? Idea was frontend input component either queries the API (using filter) or doesn't need to query at all, bc already has figured out common programs from the items?